SD_DSC.adoc

```diff
@@ -1485,15 +1485,15 @@ The evaluator shall also examine the TSS to determine the behaviour that will oc
 
 ====== AGD
 
-The evaluator shall examine the guidance documentation to ensure that instructions for configuring the number of successive unsuccessful authentication attempts and time period (if implemented) are provided, and that the process of unlocking the SDOs is described for each "action" specified (if that option is chosen). 
+If the administrator is able to configure any of the variables for authorization attempts, the evaluator shall examine the guidance documentation to ensure that instructions for configuring the number of successive unsuccessful authentication attempts and time period (if implemented) are provided, and that the process of unlocking the SDOs is described for each "action" specified (if that option is chosen). 
 
 The evaluator shall examine the guidance documentation to confirm that it describes, and identifies the importance of, any actions that are required in order to ensure that access to SDOs can be maintained, unless it is made permanently unavailable due to a factory reset.
 
 ====== Test
 
 The evaluator shall perform the following tests for each method by which the TSF authorizes access the SDOs (e.g. any passwords entered as part of establishing authorization):
 
-Test 1: The evaluator shall use the operational guidance to configure the number of successive unsuccessful authorization attempts allowed by the TOE (and, if the time period selection in FIA_AFL_EXT.1.3 is included in the ST, then the evaluator shall also use the operational guidance to configure the time period after which access is re-enabled). The evaluator shall test that once the authorization attempts limit is reached, authorization attempts with valid credentials are no longer successful.
+Test 1: If the administrator is able to configure any of the variables for authorization attempts, the evaluator shall use the operational guidance to configure the number of successive unsuccessful authorization attempts allowed by the TOE. The evaluator shall test that once the authorization attempts limit is reached (whether configured by the administrator or statically specified), authorization attempts with valid credentials are no longer successful.
```

Comment:
Same comment as above. I believe that the act of configuring the TOE should be tested as part of FMT_SMF.1

Reply:
I concur